src/components/geo/GeoViz/Region.js

```diff
@@ -59,7 +59,7 @@ const boundsFilter = ['==', '$type', 'Polygon'];
 const pointsFilter = ['==', '$type', 'Point'];
 
 const colors = [
-    '#d5e7e2',
+    '#ffffff',
```

Comment:
The server-side generated chart has a linear gradient for colors
The client-side chart uses n stops

Reply:
Should pull the color scheme from server